middlewares/memberValidation.js

```diff
@@ -0,0 +1,40 @@
+const jwt = require("jsonwebtoken");
+const User = require("../models/User");
+const ERROR_MESSAGE = require("../constants");
+
+const { ACCESS_SECRET_KEY } = process.env;
+
+const validateMember = async (req, res, next) => {
+  const { id } = req.params;
+
+  try {
+    const { accessToken } = req.cookies;
+    const { newAccessToken } = res.locals;
+
+    const { email } = newAccessToken
+      ? jwt.verify(newAccessToken, ACCESS_SECRET_KEY)
+      : jwt.verify(accessToken, ACCESS_SECRET_KEY);
```

Comment:
```suggestion
    const { accessToken } = req.cookies;
    const { newAccessToken } = res.locals;
    const { email } = newAccessToken
      ? jwt.verify(newAccessToken, ACCESS_SECRET_KEY)
      : jwt.verify(accessToken, ACCESS_SECRET_KEY);
```

제가 tokenValidation 작업할 때는 변수 선언해주는 부분을 다 붙여서 작성했는데
서영님 코드 보니까 email 작업하는 부분은 좀 긴 로직이라 이렇게 분리해주는 것도 좋은 것 같다는 생각이 들더라구요-

두 분은 어떻게 생각하시는지 궁금합니다!

(p.s 제가 작성한대로 붙여서 suggestion 걸어두었습니다-)

Reply:
저는 서영님께서 작성하신 것처럼 개행하는 게 더 가독성이 좋아보이는 것 같아요!